cclunit-framework-source/doc/examples/mocking_api.inc

```diff
@@ -0,0 +1,70 @@
+;;; put the following script definition in a .prg file and compile it
```

Comment:
Well, technically the CCLUTMOCKING.md is supposed to be the introduction for mocking.  I realize that people can read things in any order and hop around as they please, but this document links to it first, and there's a full example in there using tables.

Would it be more helpful if I said:
"These unit tests demonstrate a basic example of how the CCL Unit Testing framework's mocking API can be used instead of "with replace"."

Or if that's not acceptable, should I just move the example from CCLUTMOCKING here (or create a separate file for it (forgot to do that on the last commit) and link to it from both spots)?

This section was more based on the introduction of what mocking is than the full suite, and I was reusing your example from your blog which was just mocking the script, so my preference would be to clearly state that it's a basic example, or just reuse the CCLUTMOCKING example rather than having a separate example.

Reply:
Yes, it would make more sense if the stated purpose were "how to accomplish a basic 'with replace' while using the mocking framework" opposed to "demonstrate the mocking api".

It would also be good to modify "Details on the API can be found....." to say "Example usage and details...". Otherwise this still feels like the first offering of any example usage of the mocking api.

Reply:
Alright, I'll make those changes.